app/src/main/java/com/jerboa/ui/components/community/CommunityActivity.kt

```diff
@@ -106,127 +108,134 @@ fun CommunityActivity(
             }
         },
         content = {
-            PostListings(
-                posts = communityViewModel.posts,
-                contentAboveListings = {
-                    communityViewModel.communityView?.also { cv ->
-                        CommunityTopSection(
-                            communityView = cv,
-                            onClickFollowCommunity = { cfv ->
-                                communityViewModel.followCommunity(
-                                    cv = cfv,
-                                    account = account,
-                                    ctx = ctx,
-                                )
-                            },
-                        )
-                    }
-                },
-                onUpvoteClick = { postView ->
-                    communityViewModel.likePost(
-                        voteType = VoteType.Upvote,
-                        postView = postView,
-                        account = account,
-                        ctx = ctx,
-                    )
-                },
-                onDownvoteClick = { postView ->
-                    communityViewModel.likePost(
-                        voteType = VoteType.Downvote,
-                        postView = postView,
-                        account = account,
-                        ctx = ctx,
-                    )
-                },
-                onPostClick = { postView ->
-                    navController.navigate(route = "post/${postView.post.id}")
-                },
-                onPostLinkClick = { url ->
-                    openLink(url, ctx, appSettingsViewModel.appSettings.value?.useCustomTabs ?: true)
-                },
-                onSaveClick = { postView ->
-                    account?.also { acct ->
-                        communityViewModel.savePost(
+            siteViewModel.siteRes?.site_view?.also { siteView ->
```

Comment:
This is another reason to only pass the `enable_downvotes` setting... not having to wait for the entire siteview to get back. 

Then you can just do `siteViewModel.siteRes?.site_view?.local_site?.enable_downvotes ?: true`